.github/workflows/linter.yml

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-name: Lint
+name: pre-commit
```

Comment:
what is the significance of this change?

Reply:
It should change the button in the README to `pre-commit: passing` instead of `lint: passing`